### DISCUSSION TITLE
Error - Start the Tailwind CLI build process

### DISCUSSION THREAD

**Discussion by Acyony** [tailwindlabs/tailwindcss#7072] [Help]
Hello everybody!

I am learning how to use Tailwind CSS v3.0 and when I run the CLI tool, I got these errors.

Some properties are generated with empty values.
I did some research, but I didn't find any pieces of information that could be helpful.

Does someone know how to use Tailwind CSS v3.0?

Ps: I could use an old version in my project, but now I'm really curious to know what is the cause of this issue.

![Screenshot 2022-01-14 at 11 45 56](https://user-images.githubusercontent.com/42450397/149565300-614ff67c-0d5f-47da-8b72-cc7fd686e850.png)

**Comment by RobinMalfait** [comment]
Hey!

This is not an error at all, those empty spaces are by design to ensure that we have default values for certain css properties instead of undefined/missing css variables.
This is a bug in your editor incorrectly identifying errors.

**Reply by Acyony** [reply]
Hey Robin. Thank you so much. I took the whole day trying to understand what was the prob. 
Now I understood. =)
